apps/accounts/admin.py

```diff
@@ -1448,6 +1450,25 @@ def send_approval_email(
             template_html="emails/verification_request_approved.html",
         )
 
+    def log_message(
```

Comment:
This convenience method is intended to make it much easier to log changes we make that might not otherwise be made in the admin